Creación del componente <Card />

gabrielpindo opened the pull request:
Se creó el componente card, que será utilizado para mostrar testimonios, al staff, las actividades, etc. 
![tarjetasVista3](https://user-images.githubusercontent.com/69809704/200147309-c35a9e4b-8e17-4123-982b-0f7904cdab6e.png)
![tarjetaSinSocialMedia](https://user-images.githubusercontent.com/69809704/200150126-5edbf33e-5b89-4843-a20b-1b20518bc223.png)
![parametrosQueRecibe2](https://user-images.githubusercontent.com/69809704/200150127-a8844264-4a62-446a-83bc-4ae8fd697467.png)
![tarjetaConSocialMedia](https://user-images.githubusercontent.com/69809704/200150132-1d26f80f-1742-447f-9bac-b4d3706b8164.png)

![parametrosQueRecibe](https://user-images.githubusercontent.com/69809704/200150130-0740bba5-9033-4711-89a3-c025591fcdfe.png)

gabrielpindo committed:
Creación del componente <Card />

Componente Card, para mostrar las tarjetas del staff, actividades, testimonios, etc.

gabrielpindo committed:
Se eliminó la ruta "/" en App

gabrielpindo committed:
Creación del componente <Card/>

Se utilizará este componente para mostrar al staff, actividades, testimonios, etc.

JorgeGarciaMdz reviewed (approved):
Se ve todo bien.

gabrielpindo committed:
Eliminación de estilos del body

milandre commented on src/Components/Card/Card.jsx:
```diff
@@ -0,0 +1,63 @@
+
+            <div className="cardContainer">
+                <div className="cardImage">
+                    <img src={`${data.image}`} alt="img no disponible" />
```
Cambiar el texto del alt para que maneje tanto si hay imagen o no

gabrielpindo replied:
Ya se resolvió, muestra una imagen por defecto si no detecta una imagen en los atributos.

milandre commented on src/Components/Card/Card.jsx:
```diff
@@ -0,0 +1,63 @@
+                    </div>
+
+                    :
+                    <div style={divStyle} className="cardMedia">
```
Consulta, ¿Qué haria este div? ¿Sólo colocar el backgroundColor?

gabrielpindo replied:
Si, solo eso a modo de detalle, si no recibe los atributos de "facebook" o "linkedin". Si los recibe, muestra otro contenedor con los iconos de las redes.

milandre commented on src/Components/Card/Card.jsx:
```diff
@@ -0,0 +1,63 @@
+
+                            ?
+                            <div style={divStyle} className="cardMedia">
+                                <a href={`${data.facebook}`} target="_blank" rel='noreferrer'>
```
¿Qué pasa si las redes sociales están con string "" o null?

gabrielpindo replied:
Hay que colocar en el atributo "socialMedia=false", y muestra solo el campo vacío a color, sin los íconos de redes sociales.

gabrielpindo replied:
Se resolvió, si recibe un parametro nulo, vacío ( " " ) o indefinido, no muestra los íconos de las redes sociales.

gabrielpindo committed:
Imagen de alternativa de la etiqueta <img />

gabrielpindo committed:
Correcciones hechas en <Card />

Al no recibir link de facebook o linkedin, no mustra los iconos en la barra inferior.

gabrielpindo committed:
Update Card.jsx

gabrielpindo committed:
Correcciones de la tarjeta <Card />

gabrielpindo commented:
Buenas tardes, se realizó en el último commit pair programming para definir el comportamiento de la Card y los estilos.
